### PR TITLE
Addremovecourse

### PR DESCRIPTION
**Pull request by GitHubUser5136**
I have done:
1. Add Course
2. Remove Course
3. Update both Course List and Registered Course whenever a course is added or removed.
4. Whenver the app is restarted, the courses that are registered previously will still remain there.
5. Creating txt files for the user whenever a new user sign up.

### CODE REVIEW

File: src/sample/courseRegController.java [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 import com.jfoenix.controls.JFXButton;
 
-// import javafx.collections.FXCollections;
-// import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
 import javafx.scene.control.ListView;
 import javafx.scene.control.TextField;
-import javafx.scene.image.ImageView;
+import javafx.scene.image.ImageView;
```
Comment (Michelle-Lohwt) [inline src/sample/courseRegController.java:24]: Why there is spaces added

File: src/sample/courseRegController.java [hunk 2]
```diff
@@ -48,67 +55,169 @@ public void ContactUs(MouseEvent event) throws IOException {
...
+    try {
+      File fileObj = new File("student1courseList.txt");
+      Scanner fileReader = new Scanner(fileObj);
+      while (fileReader.hasNextLine()) {
+        courseList.getItems().add(fileReader.nextLine());
+      }
+      fileReader.close();
+    
+    } catch (FileNotFoundException e) {
+      System.out.println("An error occurred.");
```
Comment (Michelle-Lohwt) [inline src/sample/courseRegController.java:70]: Did you output the "An error occurred" to the users?

Same applies to the other "An error occurred"

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Just the error occurred thing, whether you want to show to users

File: src/sample/courseRegController.java (hunk 2 above)
Comment (GitHubUser5136) [reply src/sample/courseRegController.java:70]: No, this one only output in terminal. If want to output to the users, then need to provide a space in the app to output this error.

**Pull request merged**